### DISCUSSION TITLE
Weekly Sync #29 - Jul 21, 2021

### DISCUSSION THREAD

**Discussion by turolopezsanabria** [SUI-Components/sui-components#1615] [Weekly Sync Streaming]
[Notes from our last Weekly meeting](https://github.com/SUI-Components/sui-components/discussions/1604)

**Agenda**

- useSunscreen()
- [Dashboard](https://pages.github.mpi-internal.com/scmspain/design-systems/sui/index.html)
- Validation Code agreements
  - [demo](https://sui-components-pshno8el5-schibstedspain.vercel.app/workbench/atom/validationCode/demo)
  - [discussion](https://github.com/SUI-Components/sui-components/discussions/1574)
- Status management @beatrizip (título ilustrativo :)
- SUI-Tests metrics
	- [current status](https://github.com/SUI-Components/sui-components/tree/add/test-coverage)
	- [draft](https://github.com/SUI-Components/sui-components/pull/1621)
- AtomTooltip
	- [Demo broken](https://sui-components.vercel.app/workbench/atom/tooltip/demo)
	- [WIP PR](https://github.com/SUI-Components/sui-components/pull/1616) 
- [Contributor Docs Draft](https://github.com/SUI-Components/sui-components/pull/1611/files)
- [Documentation library](https://github.com/SUI-Components/docs-components/tree/master/components/documentation/library)
- Flexibility

**Comment by nistadev** [comment]
Getting ready for holidays... 👇🏽
```javascript
useSunscreen()
  .then(() => enjoyVacation())
```
